libs/stats/README.md

```diff
@@ -13,21 +12,199 @@ pip install --extra-index-url="https://packages.dea.ga.gov.au" odc-stats
 
 Usage
 -----
+Stats offers a set of tools to generate clear pixel count and geometric median on Sentinel-2 and Landsat satellite images and it's currently being extended to support sibling products.
 
-### Prepare tasks for processing:
+### Steps to run stats
 
-From a machine that has access to your database, run:
+### 1- Save tasks
+
+From your sandbox (or a machine that has access to your database), run:
 
 ```
-odc-stats save-tasks --year YYYY --grid  grid-name  --temporal-range temporal-range product-name
+odc-stats save-tasks --frequency annual --temporal-range 2015--P6Y --grid au-10  ga_ls8c_ard_3
 ```
-This will generate a cache file that contains tasks.  For more options, check the help.  
-### Publish tasks 
+
+The above command will generate the following files i.e. a csv file containing a list of tasks for years between 2015 to 2020, a cache file that will be used as an input to the stats cli, and several geojson files (one for each year).  
 
 ```
-odc-stats publish-tasks DB-file-path queue-name 
+ga_ls8c_ard_3_2015--P6Y-2015--P1Y.geojson  
```

Comment:
this looks broken

Reply:
Removed the frequency and updated the geojson files.